From a6bed2bc29e92430fdcccafef94f78c3e52b5f28 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 10 Dec 2019 10:53:28 +0100
Subject: [PATCH 270/529] ARM: dts: stm32: move dsi supply for stm32mp157

The supply phy-dsi is common to all boards  which use DSI
controller for stm32mp157.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: I300b0b69e1712529769a98daffe828bcc53e7ec3
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/228329
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yannick FERTRE <yannick.fertre@foss.st.com>
Tested-by: Yannick FERTRE <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/280099
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 arch/arm/boot/dts/stm32mp157.dtsi                             | 1 +
 arch/arm/boot/dts/stm32mp157a-ev1-scmi.dtsi                   | 1 -
 arch/arm/boot/dts/stm32mp157a-ev1.dts                         | 1 -
 arch/arm/boot/dts/stm32mp157a-icore-stm32mp1-ctouch2-of10.dts | 1 -
 arch/arm/boot/dts/stm32mp157a-icore-stm32mp1-edimm2.2.dts     | 1 -
 arch/arm/boot/dts/stm32mp157c-dk2-scmi.dtsi                   | 1 -
 arch/arm/boot/dts/stm32mp157c-dk2.dts                         | 1 -
 arch/arm/boot/dts/stm32mp157c-ev1-scmi.dtsi                   | 1 -
 arch/arm/boot/dts/stm32mp157c-ev1.dts                         | 1 -
 arch/arm/boot/dts/stm32mp157d-ev1.dts                         | 1 -
 arch/arm/boot/dts/stm32mp157f-dk2.dts                         | 1 -
 arch/arm/boot/dts/stm32mp157f-ev1.dts                         | 1 -
 12 files changed, 1 insertion(+), 11 deletions(-)

--- a/arch/arm/boot/dts/stm32mp157.dtsi
+++ b/arch/arm/boot/dts/stm32mp157.dtsi
@@ -20,6 +20,7 @@
 		dsi: dsi@5a000000 {
 			compatible = "st,stm32-dsi";
 			reg = <0x5a000000 0x800>;
+			phy-dsi-supply = <&reg18>;
 			clocks = <&rcc DSI_K>, <&clk_hse>, <&rcc DSI_PX>;
 			clock-names = "pclk", "ref", "px_clk";
 			resets = <&rcc DSI_R>;
--- a/arch/arm/boot/dts/stm32mp157a-ev1-scmi.dtsi
+++ b/arch/arm/boot/dts/stm32mp157a-ev1-scmi.dtsi
@@ -22,7 +22,6 @@
 };
 
 &dsi {
-	phy-dsi-supply = <&reg18>;
 	clocks = <&rcc DSI_K>, <&scmi_clk CK_SCMI_HSE>, <&rcc DSI_PX>;
 };
 
--- a/arch/arm/boot/dts/stm32mp157a-ev1.dts
+++ b/arch/arm/boot/dts/stm32mp157a-ev1.dts
@@ -291,7 +291,6 @@
 };
 
 &dsi {
-	phy-dsi-supply = <&reg18>;
 	status = "okay";
 
 	ports {
--- a/arch/arm/boot/dts/stm32mp157a-icore-stm32mp1-ctouch2-of10.dts
+++ b/arch/arm/boot/dts/stm32mp157a-icore-stm32mp1-ctouch2-of10.dts
@@ -46,7 +46,6 @@
 
 &dsi {
 	status = "okay";
-	phy-dsi-supply = <&reg18>;
 
 	ports {
 		port@0 {
--- a/arch/arm/boot/dts/stm32mp157a-icore-stm32mp1-edimm2.2.dts
+++ b/arch/arm/boot/dts/stm32mp157a-icore-stm32mp1-edimm2.2.dts
@@ -46,7 +46,6 @@
 
 &dsi {
 	status = "okay";
-	phy-dsi-supply = <&reg18>;
 
 	ports {
 		port@0 {
--- a/arch/arm/boot/dts/stm32mp157c-dk2-scmi.dtsi
+++ b/arch/arm/boot/dts/stm32mp157c-dk2-scmi.dtsi
@@ -29,7 +29,6 @@
 };
 
 &dsi {
-	phy-dsi-supply = <&reg18>;
 	clocks = <&rcc DSI_K>, <&scmi_clk CK_SCMI_HSE>, <&rcc DSI_PX>;
 };
 
--- a/arch/arm/boot/dts/stm32mp157c-dk2.dts
+++ b/arch/arm/boot/dts/stm32mp157c-dk2.dts
@@ -34,7 +34,6 @@
 
 &dsi {
 	status = "okay";
-	phy-dsi-supply = <&reg18>;
 
 	ports {
 		port@0 {
--- a/arch/arm/boot/dts/stm32mp157c-ev1-scmi.dtsi
+++ b/arch/arm/boot/dts/stm32mp157c-ev1-scmi.dtsi
@@ -27,7 +27,6 @@
 };
 
 &dsi {
-	phy-dsi-supply = <&reg18>;
 	clocks = <&rcc DSI_K>, <&scmi_clk CK_SCMI_HSE>, <&rcc DSI_PX>;
 };
 
--- a/arch/arm/boot/dts/stm32mp157c-ev1.dts
+++ b/arch/arm/boot/dts/stm32mp157c-ev1.dts
@@ -291,7 +291,6 @@
 };
 
 &dsi {
-	phy-dsi-supply = <&reg18>;
 	status = "okay";
 
 	ports {
--- a/arch/arm/boot/dts/stm32mp157d-ev1.dts
+++ b/arch/arm/boot/dts/stm32mp157d-ev1.dts
@@ -291,7 +291,6 @@
 };
 
 &dsi {
-	phy-dsi-supply = <&reg18>;
 	status = "okay";
 
 	ports {
--- a/arch/arm/boot/dts/stm32mp157f-dk2.dts
+++ b/arch/arm/boot/dts/stm32mp157f-dk2.dts
@@ -34,7 +34,6 @@
 
 &dsi {
 	status = "okay";
-	phy-dsi-supply = <&reg18>;
 
 	ports {
 		port@0 {
--- a/arch/arm/boot/dts/stm32mp157f-ev1.dts
+++ b/arch/arm/boot/dts/stm32mp157f-ev1.dts
@@ -291,7 +291,6 @@
 };
 
 &dsi {
-	phy-dsi-supply = <&reg18>;
 	status = "okay";
 
 	ports {
